README.md

```diff
@@ -21,38 +21,54 @@ Here is a little demo
 
 * Holds various resource files that may be used by SlicerCineTrack like Icon and UI elements
 
-## Quick Setup Instructions for developer
+## Setup Instructions
 
-If you want to clone this demo and make it work with 3D Slicer software, please follow steps below
+A video is attached to show sample usage and behaviour of SlicerCineTrack. This demo can also be performed locally, using the provided steps and dataset below.
 
-1) To access the module within your 3D Slicer installation you must first make the module available to 3D Slicer. Open 3D Slicer and click:
 
-`Edit` -> `Application Settings` -> `Modules`
 
-Now add the path to the `Track` folder within the `Additional module paths` selection box
+[!SlicerCineTrackTutorial.mp4](https://github.com/user-attachments/assets/d72f8d53-ca74-4ec4-a0d0-8e7131de3a62)
+
 
-* Hint: Click on the `>>` button to show the `Add` button
-* Ensure to add the path to `SlicerCineTrack/Track`
-* After the path is added, the program will restart
+
+1) Install the "SlicerCineTrack" extension from the 3D Slicer extension manager
 
 2) Reopen 3D Slicer and click on the module selection dropdown menu and find the SlicerCineTrack module under:
 
    `SlicerCineTrack` -> `Track`
 
-3) Download sample dataset
-   You can use 2 datasets below:
+3) Download a sample dataset
+   You can use either of the 2 datasets below:
 
    * [Dataset 1](https://github.com/laboratory-for-translational-medicine/SlicerCineTrack/releases/download/v1.0.0-SlicerCineTrack/Data.zip)
    * [Dataset 2](https://drive.google.com/drive/folders/1qJj53YfGM4Q7atsI-XZyySvR-F98ENXA?usp=sharing)
 
 4) Unzip the data into folder `Track/Data`
 5) Use the input selectors to import the sample data from the `Track/Data` folder:
 
-   * Cine Images Folder : `Track/Data`
+   * Cine Images : `Track/Data/2D Cine Images`
    * Segmentation File : `Track/Data/Segmentation.mha`
    * Transforms File : `Track/Data/Transforms.csv`
 
-6) Press the `Play` button to begin playback
+6) Press the `Play` button to begin playback 
+
+## Using the Preview Release of SlicerCineTrack for Development
+
```

Comment:
I have given development instructions here @haphantran. If you want me to add further instructions, I can add those here

Reply:
Sorry I overlooked that part. LGTM. Thanks.